integration_test/third_party_apps_data/applications/hbase/centos_rhel/install

```diff
@@ -0,0 +1,36 @@
+set -e
+
+sudo yum -y install \
+    java-1.8.0-openjdk java-1.8.0-openjdk-devel wget
+
+sudo mkdir /opt/hbase
+wget http://apache.mirror.gtcomm.net/hbase/stable/hbase-2.4.9-bin.tar.gz
+sudo tar -xvf hbase-2.4.9-bin.tar.gz -C /opt/hbase --strip 1
+
+JAVA_HOME="$(dirname "$(dirname "$(readlink "$(readlink "$(which javac)")")")")"
+
+cat <<EOF | sudo tee /etc/systemd/system/hbase.service
+[Unit]
+Description=HBase Hadoop database
+After=network.target
+
+[Service]
+Type=forking
+Environment=JAVA_HOME=$JAVA_HOME
+ExecStart=/opt/hbase/bin/start-hbase.sh
+ExecStop=/opt/hbase/bin/stop-hbase.sh
+User=root
+Group=root
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+cat <<EOF | sudo tee /opt/hbase/conf/hbase-env.sh
+export HBASE_JMX_BASE="-Djava.rmi.server.hostname=0.0.0.0 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false"
+export HBASE_MASTER_OPTS="\$HBASE_MASTER_OPTS \$HBASE_JMX_BASE -Dcom.sun.management.jmxremote.rmi.port=9999 -Dcom.sun.management.jmxremote.port=9999"
```

Comment:
You have port 9999 but the sample config in the upstream `hbase-env.sh` sets 10101; we should use the default port.

Also, the comments in the sample config recommend using a different mechanism to configure JMX: https://hbase.apache.org/book.html#jmx_config

Reply:
[Here](https://hbase.apache.org/book.html#_get_started_with_hbase) it also discusses setting up the environment variables in this [file](https://github.com/apache/hbase/blob/master/conf/hbase-env.sh) which already has commented out options in it. it seemed cleaner to do it this way. Also, most of the top results from a quick google search shows using environment variables. So, if we're trying to replicate what most users would do I think this would be it.

Reply:
Yeah, I don't know which of the two ways to configure JMX is better, but I do know you're not using the default port here - why not?

(The file you linked contains:
```
# NOTE: HBase provides an alternative JMX implementation to fix the random ports issue, please see JMX
# section in HBase Reference Guide for instructions.
```
That's what I was referencing with the link to #jmx_config)

Reply:
I defer to your HBase knowledge. I've never used HBase so I don't know if one of the JMX configurations is more popular than the other. At some point it would be great if we could test both, but we don't have a good way to do that right now.